Apps/Sandcastle/gallery/3D Tiles Adjust Height.html

```diff
@@ -56,29 +56,27 @@
 var toolbar = document.getElementById('toolbar');
 Cesium.knockout.applyBindings(viewModel, toolbar);
 
-Cesium.CesiumIon.create3DTileset(3883, { accessToken: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiJjYjNlMTg5Zi1hMzc2LTRmZjktOGEwZC00NGEzNTM0MTAzZGUiLCJpZCI6MjU5LCJpYXQiOjE1MTgxOTE2NTV9.qaP8-_Ej6AihGnv5iB990Hm6lHr8F_rrC3_EPxdT6MQ' })
+var tileset = new Cesium.Cesium3DTileset({ url: Cesium.CesiumIon.createResource(3883) });
+tileset.readyPromise
     .then(function(tileset) {
```

Comment:
Put `.then` on same line.

Reply:
I prefer `then` on the next line.  I think it looks cleaner when you have multiple `then` and/or `otherwise` blocks

Reply:
While I agree with @hpinkos, current Cesium style is to put them on the same line (and the debate over it is well outside the scope of this PR), so I'll change it.